### PR TITLE
replaced footer with Identifier component for compliance score

### PR DESCRIPTION
**Pull request by nick-mon1**
For more details see https://trello.com/c/K1r012YQ

### CODE REVIEW

File: _includes/footer.html [hunk 1]
```diff
@@ -5,53 +5,82 @@
...
-        <div class="separator display-flex flex-align-center">
-          <a href="https://digital.gov"
-            ><img
-              class="height-3 display-block"
-              src="{{ '/assets/img/digital-gov-logo--white.png' | relative_url }}"
-              alt="Digital.gov"
-          /></a>
+
+
+  <div class="usa-identifier">
```
Comment (mejiaj) [inline _includes/footer.html:10]: Can we move this to its own include since it's a separate component?

File: _includes/footer.html [hunk 2]
```diff
@@ -5,53 +5,82 @@
...
+    <section class="usa-identifier__section usa-identifier__section--masthead" aria-label="Agency identifier">
+      <div class="usa-identifier__container">
+        <div class="usa-identifier__logos">
+          <a href="https://www.gsa.gov" class="usa-identifier__logo">
+            <img class="usa-identifier__logo-img" src="{{ '/assets/img/gsa-logo-w100.png' | relative_url }}" alt="GSA logo" role="img">
```
Comment (mejiaj) [inline _includes/footer.html:15]: Is there an SVG version of the logo available? Would look better on higih res screens.

**Review by mejiaj — CHANGES_REQUESTED** [review]
Thanks Nick, visually it looks great. Added some suggestions to make it easier to update in the future and improve logo quality on high-res screens.

File: _includes/footer.html (hunk 2 above)
Comment (nick-mon1) [reply _includes/footer.html]: Still looking for a high-res version...

File: _includes/footer.html (hunk 1 above)
Comment (nick-mon1) [reply _includes/footer.html]: Moved to _includes/identifier.html

**Review by mejiaj — CHANGES_REQUESTED** [review]
Looking better, added additional comments. Thanks Nick!

**Comment by mejiaj** [comment]
The previous component also had the background color of `#0b4778`, we might want to consider a darker shade of that blue. In this case the `#0b4778` is too light and makes the GSA logo hard to see.


localhost_4000_do-i-need-clearance_

File: _includes/identifier.html [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+        <p class="usa-identifier__identity-disclaimer">
+          An official website of the <a href="https://www.gsa.gov/" class="usa-link">U.S. General Services Administration</a> and
+          the <a href="https://www.whitehouse.gov/omb/">Office of Management and Budget</a>
+        </p>
+      </div>
+    </div>
+  </section>
+  <nav
+    class="usa-identifier__section usa-identifier__section--required-links"
+    aria-label="Important links,,,"
```
Comment (mejiaj) [inline _includes/identifier.html:35]: Please remove extra comma's

File: _includes/identifier.html [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+          <a href="https://pra.digital.gov/privacy" class="usa-identifier__required-link usa-link">
+            Privacy policy
+          </a>
+        </li>
+      </ul>
+    </div>
+  </nav>
+  <section
+    class="usa-identifier__section usa-identifier__section--usagov"
+    aria-label="U.S. government information and services,,,"
```
[the remark this answers is not in the document]
Comment (nick-mon1) [reply _includes/identifier.html]: Removed

File: _includes/identifier.html (hunk 3 above)
Comment (nick-mon1) [reply _includes/identifier.html]: Removed

File: _scss/components/_identifier.scss [hunk 5]
```diff
@@ -0,0 +1,23 @@
+.usa-identifier {
+  font-size: 1rem;
+  line-height: 1.5;
+}
+
+@include at-media("tablet") {
```
[the remark this answers is not in the document]
Comment (nick-mon1) [reply _scss/components/_identifier.scss]: Removed

File: _scss/components/_footer.scss [hunk 6]
```diff
@@ -1,3 +1,12 @@
...
+}
+
+.footer-contact {
+  font-size: 1rem;
+  line-height: 1.5;
+}
+
 .usa-footer__secondary-section,
 .usa-footer__secondary-section a,
 .usa-footer__secondary-section a:hover,
```
[the remark this answers is not in the document]
Comment (nick-mon1) [reply _scss/components/_footer.scss]: Updated

File: _includes/footer.html [hunk 7]
```diff
@@ -5,53 +5,16 @@
...
-      </div>
+      <b>Contact:</b>
+      <a href="https://pra.digital.gov/contact/" class="text-ink text-bold hover:text-no-underline">Get PRA Help</a> |
+      <a href="invalid-mailto:%70%72%61@%6F%6D%62.%65%6F%70.%67%6F%76?subject=PRA%20Website:%20Site%20Feedback&body=Please%20fill%20out%20the%20information%20below%0A%0A%0AWhat%20were%20you%20trying%20to%20do?%0A%0A%0AGeneral%20Comments" class="text-ink text-bold hover:text-no-underline">
+        Email us site feedback
+      </a>
     </div>
   </div>
+
+  {% include identifier.html %}
```
[the remark this answers is not in the document]
Comment (nick-mon1) [reply _includes/footer.html]: Now moved to `./_layouts/default.html`

**Review by afeijoo — CHANGES_REQUESTED** [review]
1. The two logos should link to their respective homepages.
2. The `Email us site feedback` link starts with an "invalid" code:

[Email us site feedback](invalid-mailto:%70%72%61@%6F%6D%62.%65%6F%70.%67%6F%76?subject=PRA%20Website:%20Site%20Feedback&body=Please%20fill%20out%20the%20information%20below%0A%0A%0AWhat%20were%20you%20trying%20to%20do?%0A%0A%0AGeneral%20Comments)

**Review by afeijoo — APPROVED** [review]

**Review by mejiaj — APPROVED** [review]
Components and links look good, no issues found. Thanks Nick!

**Pull request merged**